diff --git a/dist/index.d.ts b/dist/index.d.ts
index 478779590393dc4cf07d60e72ee8f41e31b69fdb..01c3cc9e82203fa5487f24e5a10bdba565f838e4 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -69,8 +69,13 @@ interface AuthorInfo {
     email: string;
     name: string;
 }
+interface GitHubRelease {
+    html_url: string
+    id: number
+    upload_url: string
+}
 
-declare function sendRelease(options: ChangelogOptions, content: string): Promise<void>;
+declare function sendRelease(options: ChangelogOptions, content: string): Promise<GitHubRelease>;
 declare function resolveAuthorInfo(options: ChangelogOptions, info: AuthorInfo): Promise<AuthorInfo>;
 declare function resolveAuthors(commits: Commit[], options: ChangelogOptions): Promise<AuthorInfo[]>;
 declare function hasTagOnGitHub(tag: string, options: ChangelogOptions): Promise<boolean>;
@@ -96,4 +101,4 @@ declare function resolveConfig(options: ChangelogOptions): Promise<Required<Chan
 
 declare function parseCommits(commits: RawGitCommit[], config: ChangelogenOptions): GitCommit[];
 
-export { AuthorInfo, ChangelogOptions, ChangelogenOptions, Commit, GitHubAuth, GitHubRepo, ResolvedChangelogOptions, defineConfig, generate, generateMarkdown, getCurrentGitBranch, getFirstGitCommit, getGitHubRepo, getLastGitTag, hasTagOnGitHub, isPrerelease, isRefGitTag, isRepoShallow, parseCommits, resolveAuthorInfo, resolveAuthors, resolveConfig, sendRelease };
+export { AuthorInfo, ChangelogOptions, ChangelogenOptions, Commit, GitHubAuth, GitHubRepo, GitHubRelease, ResolvedChangelogOptions, defineConfig, generate, generateMarkdown, getCurrentGitBranch, getFirstGitCommit, getGitHubRepo, getLastGitTag, hasTagOnGitHub, isPrerelease, isRefGitTag, isRepoShallow, parseCommits, resolveAuthorInfo, resolveAuthors, resolveConfig, sendRelease };
diff --git a/dist/shared/changelogithub.821fab93.mjs b/dist/shared/changelogithub.821fab93.mjs
index 5fc2100867613c20f7827eac8715a5fc28bdc39e..2c7e9968f1e5b9e7b84f1b4a44e52f5467e3edab 100644
--- a/dist/shared/changelogithub.821fab93.mjs
+++ b/dist/shared/changelogithub.821fab93.mjs
@@ -32,6 +32,7 @@ async function sendRelease(options, content) {
     headers
   });
   console.log(green(`Released on ${res.html_url}`));
+  return res;
 }
 function getHeaders(options) {
   return {
diff --git a/dist/shared/changelogithub.a0384015.cjs b/dist/shared/changelogithub.a0384015.cjs
index e42ebc572079b1c0bc12858e1ad22d0e5340f465..a89f4359773b5c8d937fac0c470f89401c473c55 100644
--- a/dist/shared/changelogithub.a0384015.cjs
+++ b/dist/shared/changelogithub.a0384015.cjs
@@ -34,6 +34,7 @@ async function sendRelease(options, content) {
     headers
   });
   console.log(kolorist.green(`Released on ${res.html_url}`));
+  return res;
 }
 function getHeaders(options) {
   return {